docs/GettingStartedDocs/buildandsign.md

```diff
@@ -97,9 +97,37 @@ NumHeapPages=1024
 NumStackPages=1024
 NumTCS=1
 ```
+Additionally, a developer can specify additional Key Sharing and Separation (KSS) identity properties
+for use on the platforms that support it:
+- **FamilyID**: The product family identity (ISVFAMILYID) a developer can specify to group
```

Comment:
I see terms like "ISVFAMILYID" which look to be SGX-only, or are those names intended to be TEE-agnostic?  I ask because elsewhere in this PR I see "OE_ENCLAVE_FLAG_KSS" (as opposed to "OE_ENCLAVE_FLAG_SGX_KSS") so assume KSS is being defined as a TEE-agnostic concept, which any TEE might or might not support.  If so, then the descriptions of each of these fields should not be SGX specific, but can use SGX in examples, like "(e.g., ISVFAMILYID for SGX)".

Reply:
Updated